index.html

```diff
@@ -77,6 +84,30 @@ <h4>Multi-Post Stories <br />Gain+Glory</h4>
             <button class="card-button">See Projects</button>
           </div>
         </div>
+        <div class="card">
+          <div class="bottom-card">
+            <h4>Multi-Post Stories <br />Gain+Glory</h4>
+            <ul>
+              <div class="top-card"></div>
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button class="card-button">See Projects</button>
```

Comment:
- [ ] Kindly add all button interactions as shown in Figma 👍 

![image](https://user-images.githubusercontent.com/50721479/206261132-83370b75-f2b5-46ee-8489-0200e6bb6bd2.png)